exercise-3/styles.css

```diff
@@ -0,0 +1,65 @@
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.wrapper {
+  background-color: #413E3C;
+  display: flex;
+  flex-direction: row;
+  height: 30rem;
+  justify-content: center;
+  margin-top: 5rem;
+  width: 75rem;
+}
+
+input {
+  position: relative ;
+  top: 25rem;
+}
+
+labellll {
+  display: none;
+  position: fixed;
+}
+
+label {
+  padding-top: 5rem;
+  position: absolute;
+  left: -200rem;
+}
+
+input:checked + label {
+  display: block;
+  transition: 0.25s;
+  left: 31rem;
```

Comment:
This gets the job done, but it’s always better to use `transform: translate()` rather than animating basically anything else `left`, `top`, `margin`, etc. This article breaks it down pretty well: https://www.paulirish.com/2012/why-moving-elements-with-translate-is-better-than-posabs-topleft/

TL;DR you’ll get stutter and < 60FPS